File: SampleSizeApi/SampleSizeApi/Models/Request/CalculateRequest.cs
using SampleSizeApi.Models.DTO;

namespace SampleSizeApi.Models.Request
{
    public class CalculateRequest
    {
        public Input InputRequest { get; set; }
        public Pagination Pagination { get; set; } = new Pagination() { Page = 1, Next = 1 };
    }
}

File: SampleSizeWeb/src/app/model/Request/CalculateRequest.ts

import { InputModel } from "../Domain/InputModel";
import { Pagination } from "../Domain/PaginationModel";

export class CalculateRequest{
    InputRequest:InputModel | undefined;
    Pagination:Pagination | undefined;

    constructor(input?:InputModel, pagination?:Pagination){
        this.InputRequest = input;
        this.Pagination = pagination;
    }

    SetParametros(input?:InputModel, pagination?:Pagination):void{
        this.InputRequest = input;
        this.Pagination = pagination;
    }

}